### PR TITLE
[14.0][MIG] stock_barcodes_supplierinfo: Migration to 14.0

### PR DESCRIPTION
**Pull request by KNVx**
Migration to 14.0

Please @eantones can you review it?

### CODE REVIEW

**Comment by eantones** [comment]
LGTM!

File: stock_barcodes_supplierinfo/tests/test_stock_barcodes_supplierinfo.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Copyright NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestStockBarcodesSupplierInfo(TransactionCase):
```
Comment (rousseldenis) [inline stock_barcodes_supplierinfo/tests/test_stock_barcodes_supplierinfo.py]: Better to use SavepointCase and setUpClass()

File: stock_barcodes_supplierinfo/wizard/__init__.py [hunk 2]
```diff
@@ -1 +1,3 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
```
Comment (rousseldenis) [inline stock_barcodes_supplierinfo/wizard/__init__.py]: No need licensing in __init__ files

File: stock_barcodes_supplierinfo/wizard/stock_barcodes_read.py [hunk 3]
```diff
@@ -1,21 +1,23 @@
+# Copyright 2022 NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (rousseldenis) [inline stock_barcodes_supplierinfo/wizard/stock_barcodes_read.py]: You are mixing LGPL and AGPL. I think the module manifest is LGPL. Could you modify ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by eantones** [comment]
@rousseldenis all requested changes done!
cc @KNVx

**Review by fshah-initos — APPROVED** [review]
Code review and functional tests.
Looks good to me.

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@eLBati This one is LGPL but cannot depends on AGPL module.

**Pull request closed without merging**